layouts/page/cudf-polars.html

```diff
@@ -0,0 +1,307 @@
+        </div>
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-light fa-boxes-stacked"></i> Seamless CPU Fallback</h2>
+          <p>For queries whose one or more operations are unsupported on the cuDF backend, the entire query execution would gracefully fallback to the CPU.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-light fa-bolt-lightning"></i> Query Optimizer</h2>
+          <p>The cuDF backend utilizes Polars' query optimizer to ensure efficient execution and minimal memory usage.</p>
+        </div>
+        <div class="col-md-4 py-3">
+          <iframe style="max-width: 100%;" width="500" height="320"
+            src="https://www.youtube.com/embed/kmj1QOY71Ps?si=ToajeDGStxJynNdl" title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen></iframe>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+<div class="bottom-slant-container">
+  <div class="bottom-slant-down bg-white"></div>
+</div>
+
+<!-- Charts -->
+<section class="row section padding-top-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="mb-4"> RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries </h1>
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Performance improves as <br> data scales </h2>
+          {{ with .Resources.GetMatch "polars-collect.png" }}
+          <img src="{{ .RelPermalink }}">
+          {{ end }}
+          <br>
+          <p class="fs-7 fw-lighter text-center">PDS-H benchmark | GPU: NVIDIA H100 | CPU: Intel Xeon W9-3495X (Sapphire Rapids) | Storage: Local NVMe</p>
+        </div>
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-diagram-subtask"></i>  Higher acceleration with Compute vs memory and I/O bound queries </h2>
+            {{ with .Resources.GetMatch "polars-gpu-cpu.png" }}
+            <img src="{{ .RelPermalink }}">
+            {{ end }}
+            <br>
+            <p class="fs-7 fw-lighter text-center">PDS-H is derived from TPC-H but these results are not comparable to TPC-H results. <a href="#" target="_blank">Learn More <i class="fa-solid fa-arrow-up-right"></i> </a></p>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+
+<!-- Get Started -->
+ <div id="get-started"></div>
+<section class="row section padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="mb-4"> How It Works</h1>
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Default GPU </h2>
+          <p>First, install the cuDF backend for Polars with a feature flag in your standard Python pip install command:</p>
```

Comment:
This step in common in both default and custom configurations. Any chance we could keep the pip installation command just right after "How it works" section? Something like https://docs.google.com/document/d/1h2Ynp0sniNuQ1k7Cfx64DYauaLJnadIkVpWCrXudaK0/edit?